.github/workflows/ci.yaml

```diff
@@ -488,7 +507,9 @@ jobs:
       - name: Run hassfest
         run: |
           . venv/bin/activate
-          python -m script.hassfest --requirements --action validate
+          python -m script.hassfest --requirements --action generate
+          git diff
+          git diff --quiet
```

Comment:
We don't need the output here and not the diff either.
This script needs to be run locally (as the PR template suggest as well).

This also would increase "drive-by" PRs for people making patches inside GitHub, instead of setting up an actual development environment.

For those reasons, I rather not have this change committed. Please revert.

Reply:
Sigh. I had a lot of trouble convincing this to run locally and it was actually incredibly helpful as changes went.

That said, I'll drop whatever is requested, my goal is to improve whatever will be accepted.

Reply:
>  I had a lot of trouble convincing this to run locally

What troubles did you run into? Should we fix/address that instead?
Please note; that `pre-commit` runs this step for you automatically on making a new commit.

Reply:
Unresolved this review comment, as the question wasn't answered.

We should fix issues, could you explain what the issue is you are trying to address?